string model_name
string mode
---
bool ok